index.html

```diff
@@ -174,25 +230,44 @@ <h2 class="long-text">Multi-Post Stories</h2>
                     2015
                 </span>
             </p>
-            <p>
+            <p class="subtitle-desktop-card4">
+                UBER
+                <span class="grey">
+                    <img src="./img/period.svg" alt="Grey period in text">
+                    Lead Developer
+                    <img src="./img/period.svg" alt="Grey period in text">
+                    2018
+                </span>
+            </p>
+            <p class="content-card4">
```

Comment:
- [ ] Your project cards look great and the button hover effect works just as expected, well done 👏🏾; But the card container also has a hover effect on it. In the Figma image below 👇🏾 , I have placed three labels `A`, `B`, and `C`. The `A` mark shows the cursor pointer hovering on the project card, and the `B` mark shows the blue border and shadow effect on it as a result of the mouse hover. Then `B` mark is the normal card when the mouse is NOT hovering on it. 

Kindly implement this.

Figma Design
![image](https://user-images.githubusercontent.com/84156714/236068236-13a2f280-1a1f-44e3-9172-efbdde8a601a.png)